src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(String color, String name) {
+        setName(name);
+        setColor(color);
+    }
+
+    public int getFirstSide() {
+        return firstSide;
+    }
+
+    public void setFirstSide(int firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public int getSecondSide() {
+        return secondSide;
+    }
+
+    public void setSecondSide(int secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {

```

Comment:
Method 'getArea' should be implemented to calculate the area of the rectangle, not to call 'super.getArea()'.